### PR TITLE
Closes #31 Updates to reproduce Efficacy TLF from pilot1.

### LINKED ISSUE #31: Table : Primary Endpoint Analysis: Glucose (mmol/L) - Summary at Week 20 - LOCF
Once #3  is complete. Reproduce this table. See link for Pilot 1 TLG report in #6

### PR DESCRIPTION
**Pull request by DeclanHodges**
Leaving this as a draft PR for now, as code needs to be updated to use pilot3 ADLBC once ready (currently using pilot1 ADLBC).

### CODE REVIEW

**Comment by laxamanaj** [comment]
Hi, @DeclanHodges .  Just wanted to let you know that ADLBC has now been merged to `main`.  Hope you can rerun this table pointing to the pilot 3 ADLBC and make this a formal PR.  Thanks again!

Joel

**Comment by DeclanHodges** [comment]
Hi @laxamanaj, thanks for letting me know. The code and RTF have been updated to use the Pilot 3 ADLBC now, so this PR is ready for review.

File: vignettes/ectd/tlf-efficacy.R [hunk 1]
```diff
@@ -147,8 +146,15 @@ tbl_1 <- apr0ancova1 %>%
       "CI = Confidence Interval, LS = Least Squares, SD = Standard Deviation"
     )
   ) %>%
+  rtf_footnote(
+    footnote = c(
+      paste("Table generated on:", Sys.time())
+    ),
+    as_table = FALSE,
+    text_justification = "c"
+  ) %>%
```
Comment (DeclanHodges) [inline vignettes/ectd/tlf-efficacy.R:155]: @kaz462 feel free to use this code to add the run date footnote to your table.

Comment (kaz462) [reply vignettes/ectd/tlf-efficacy.R:155]: Thanks, @DeclanHodges !

Comment (kaz462) [reply vignettes/ectd/tlf-efficacy.R:155]: @laxamanaj I just realized the timestamp was already available for `tlf-primary.r`, so no further updates are needed

Comment (laxamanaj) [reply vignettes/ectd/tlf-efficacy.R:155]: Thanks, @DeclanHodges and @kaz462 !  Good to know.  Update looks great!

**Review by laxamanaj — APPROVED** [review]
Thank you, @DeclanHodges .  Looks good!  Thanks for the addition of the timestamp.  The results still match that of Pilot1 outputs.  Good job.

**Pull request merged**